src/games/gameitem.py

```diff
@@ -209,11 +261,27 @@ def _game_observers(self, game):
         return [game.to_player(name) for name in game.observers
                 if game.is_connected(name)]
 
+    def _game_sim_mods(self, game):
+        if not game.sim_mods:
+            return None
+        inst = modvault.getInstalledMods()
+        uids = [mod.uid for mod in inst]
+        mods = ([], [])
+        for uid in game.sim_mods:
+            if uid in uids:
+                mods[0].append(game.sim_mods[uid])
+            else:
+                mods[1].append(game.sim_mods[uid])
+            mods[0].sort()
+            mods[1].sort()
```

Comment:
Why are you sorting both lists after any insertion into either list? Just sort the lists at the end, surely?!